ergo-wallet/src/main/java/org/ergoplatform/wallet/interface4j/SecretString.java

```diff
@@ -27,16 +38,23 @@
 
```

Comment:
All method, not only getData, must raise an exception. 
Imagine, instead of erase we did actual de-allocation and cleanup of the memory.
Calling isEmpty would mean segmentation fault.
We cannot de-allocate, but we can make all methods throw exception.

The only method which would be allowed on erased object is `isErased`, which can be added and be used
to detect the status of SecretString.

Reply:
@ccellado What about adding `isErased` method?
I can imaging it can be useful in situation when you cannot statically know whether the secret was erased or not by the previous code.
So you have an option of either creating a new secret string, or just reusing existing, in the case it was not erased.

Reply:
Forgot to add. Fixed!